zulip_bots/zulip_bots/tests/test_finder.py

```diff
@@ -0,0 +1,19 @@
+import os
+
+from zulip_bots import finder
+
+from unittest import TestCase
+from unittest.mock import patch
+
+
+class FinderTestCase(TestCase):
+
+    def test_resolve_bot_path(self) -> None:
+        with patch('os.path.isfile', return_value=False):
```

Comment:
Why do we need to patch os.path.isfile?

Reply:
Because if a file exists at the given `name`, then the absolute path to it is returned by [`resolve_bot_path`](https://github.com/zulip/python-zulip-api/pull/401/commits/e41af88c30cb6db9acdfac5200558761c5e632e7#diff-7d6ba3f111cf7a3c9528de51bd6a1e1eR28). (We have it like this to support running custom bots i.e bots which are not included in python-zulip-api.)

So I thought we would be explicit here and patch isfile to be false to run the test for the path of an existing bot.

Reply:
In that case I think we should just remove this line. We expect `resolve_bot_path` to do the right thing on its own, and it should just see that there is no `helloworld` file in the `zulip_bots/zulip_bots` directory.

Reply:
@roberthoenig I have removed the patch for os.path.isfile .